### DISCUSSION TITLE
Border get transparent on overflow scroll

### DISCUSSION THREAD

**Discussion by jonnywintech** [tailwindlabs/tailwindcss#15495] [Help]
for some reason border bottom gets invisible on scroll x 

here is a link to my pen

https://codepen.io/jonnywintech/pen/vEBJmqr

**Comment by wongjn** [comment]
You could consider wrapping the scrolling content in another element with `width: max-content`. This makes the child elements expand the full width of the widest element: https://play.tailwindcss.com/KIwf7h4Z9R
